### DISCUSSION TITLE
Supported Platforms for race flag

### DISCUSSION THREAD

**Discussion by PG2000** [onsi/ginkgo#1432] [General]
Hello,

im wondering why darwin/arm64 is [not listed](https://github.com/onsi/ginkgo/blob/b69c00d4ce8b2d0bb408b8cf6dc3317e6a9deeb1/types/config.go#L515) here.

As stated in https://go.dev/doc/articles/race_detector go seems to have support for detecting race conditions also on darwin/arm64.

Does ginkgo use another mechanism to detect race condtions?

**Comment by onsi** [comment]
No it's just using race (and i use it all the time on my M1 Max machine so... lol).  I think I copied and pasted the docs from go test's output a few years ago.  They've added arm support since then.

If you're up for a quick PR that would be great thanks!
